src/Text/Pandoc/Writers/HTML.hs

```diff
@@ -1062,8 +1062,11 @@ inlineToHtml opts inline = do
       case writerHTMLMathMethod opts of
            WebTeX url -> do
               let imtag = if html5 then H5.img else H.img
+              let s = case t of
+                           InlineMath  -> "\\textstyle "
+                           DisplayMath -> ""
```

Comment:
Prefix display formulas in case an alternative service uses inline style by default?

Reply:
This probably makes sense.